docs/src/components/buttons/SidebarCategorizationButton.js

```diff
@@ -0,0 +1,33 @@
+// @flow strict
+import React, { type Node } from 'react';
+import { Flex, IconButton, Tooltip } from 'gestalt';
+// eslint-disable-next-line import/no-relative-parent-imports
+import { type SidebarOrganisedBy } from '../navigationContext.js';
+import trackButtonClick from './trackButtonClick.js';
+
+type Props = {|
+  onClick: () => void,
+  sidebarOrganisedBy: SidebarOrganisedBy,
+|};
+
+export default function SidebarCategorizationButton({ onClick, sidebarOrganisedBy }: Props): Node {
+  const sidebarOrganisedByCopy =
+    sidebarOrganisedBy === 'categorized' ? 'Alphabetical' : 'Categorize';
+
+  return (
+    <Tooltip inline text={`Sidebar: ${sidebarOrganisedByCopy}`}>
```

Comment:
I feel these Tooltips are not following best practices.

```
    <Tooltip inline text={`Toggle sidebar categorization from ${sidebarOrganisedByCopy} to ${sidebarOrganisedBy === 'categorized' ? 'Categorize' : 'Alphabetical'}`}>
        <IconButton
          accessibilityLabel=""
```

but I'd love to hear @ayeshakmaz thoughts,

Reply:
That's possible, but out of scope for this PR. This is just adding tracking

Reply:
There's a bug filed for Tooltips and A11Y that will affect this anyway, so worth waiting until that is fixed

Reply:
@ayeshakmaz what's the bug number?

Reply:
Also, we recommend that in best practices... so not quite sure how bug + best practices intersect.

Reply:
[PDS-2543 ](https://jira.pinadmin.com/browse/PDS-2543)